react/test/containers/ReviewsContainer.js

```diff
@@ -19,4 +19,7 @@ describe('ReviewsContainer', () => {
       }]
     });
   });
+  it('should render a ReviewForm Component with the specific props', () => {
+    expect(wrapper.find('ReviewForm')).toBePresent();
+  })
```

Comment:
If this test says it's looking for specific props, you should probably test and see if those props are there.